docs/en/ingest-management/security/enrollment-tokens.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[fleet-enrollment-tokens]]
+= {fleet} enrollment tokens
+
+An enrollment token is an {es} API key that you use to enroll one or more
+{agent}s in {fleet}. The enrollment token enrolls the {agent} in a specific
+agent policy that defines the data to be collected by the agent. You can
+use the token as many times as required. It will remain valid until you revoke
+it.
+
+The enrollment token is used for the initial communication between {agent} and
+{fleet-server}. After the initial connection request from the {agent},
+the {fleet-server} passes two API keys to the {agent}:
+
+* An output API key
++
+This API key is used to send data to {es}. It has the minimal permissions needed
+to ingest all the data specified by the agent policy. If the API key is invalid,
+the {agent} stops ingesting data into {es}.
+
+* A communication API key
++
+This API key is used to communicate with the {fleet-server}. It has only the
+permissions needed to communicate with the {fleet-server}. If the API key is
+invalid, {fleet-server} stops communicating with the {agent}.
+
+// TODO: Convert the following steps to 7.14 after this topic is merged and
+// backported to 7.13.
+
+//QUESTION: Do we want to document the API calls for creating enrollment tokens
+//or wait until we formally publish the API docs?
+
+[discrete]
+[[create-fleet-enrollment-tokens]]
+== Create enrollment tokens
+
+Create enrollment tokens and use them to enroll {agent}s in specific policies.
+
+TIP: When you use the {fleet} UI to add an agent or create a new policy, {fleet}
+creates an enrollment token for you automatically.
+
+To create an enrollment token:
+
+. Log in to {kib} and go to *Management* > *{fleet}*.
++
+[role="screenshot"]
+image::images/kibana-fleet-start.png[{fleet} in {kib}]
+
+. In {fleet}, click *Agents > Enrollment tokens > Create enrollment token*. Name
+your token and select an agent policy.
++
+[role="screenshot"]
+image::images/create-token.png[Enrollment tokens tab in {fleet}]
+
+. Click *Create enrollment token*.
+
+. In the list of tokens, click the *Show token* icon to see the token secret.
++
+[role="screenshot"]
+image::images/show-token.png[Enrollment tokens tab with Show token icon highlighted]
+
+All {agent}s enrolled through this token will use the selected policy unless you
+assign or enroll them in a different policy.
+
+To learn how to install {agent}s and enroll them in {fleet}, refer to
+<<elastic-agent-installation>>.
+
+[discrete]
+[[revoke-fleet-enrollment-tokens]]
+== Revoke enrollment tokens
+
+To revoke an enrollment token:
+
+. In {fleet}, click *Agents > Enrollment tokens*.
+
+. Find the token you want to revoke in the list and click the *Revoke token*
+icon.
++
+[role="screenshot"]
+image::images/revoke-token.png[Enrollment tokens tab with Revoke token highlighted]
+
+. Click *Revoke enrollment token*. You can no longer use this token to enroll
+{agent}s.
+
+Any {agent}s enrolled through the token will stop communicating with
```

Comment:
@nchaulet Is this the case? If the enrollment token is invalidated also all the ouput and communication keys related to it are invalidated? If yes, we should validate if this is the expected behaviour.

Reply:
No it's not the case enrolled agent will continue to work

Reply:
@nchaulet My agent definitely stopped working, but I didn't look at the logs closely to see what was going on. Might have been a weird coincidence. Should I retest, or are you sure that the agent continues to work?

Reply:
I've removed the sentence. If it turns out to be correct, I can add it back in.